rho/postprocessing.py

```diff
@@ -323,7 +323,8 @@ def process_id_u_jboss(fact_names, host_vars):
     # nonzero error code means "not found", because then we would give
     # false negatives if the user didn't have permission to read
     # /etc/passwd (or other errors).
-    if output['stdout_lines'] == ['id: jboss: no such user']:
+    if (output['stdout_lines'] == ['id: jboss: no such user'] or
```

Comment:
I would recommend doing `output['stdout_lines'].lower() == ['id: jboss: no such user']` or even `output.get('stdout_lines', '').lower() == ['id: jboss: no such user']` if there is a chance of stdout_lines being `None`.